app/assets/javascripts/modules/LoadTime.js

```diff
@@ -0,0 +1,52 @@
+// The following script sends the page load time per page to GA
+
+moj.Modules.LoadTime = {
+
+	labels: {
+		excellent: 'Under 1 second (Excellent)',
+		veryGood: '1 to 2 seconds (Very good)',
+		acceptable: '2 to 3 seconds (Acceptable)',
+		improve: '3 to 5 seconds (Try improving)',
+		fix: '5 to 8 seconds (Needs fixing)'
+	},
+	category: 'Page load time',
+
+	init: function() {
+		window.onload = this.pageLoaded.bind(this);
+	},
+
+	pageLoaded: function() {
+		if(typeof 'ga' === 'undefined')return;
+		
+		var loadTime = this.getLoadTimeSeconds();
+		if(!loadTime || isNaN(loadTime))return;
+		
+		var pageName = this.getPageName();
+		if(!pageName)return;		
+		
+		var label = this.labelLoadTime(loadTime);
+		
+		this.sendEventToGoogle(pageName, label, loadTime);
+	},
+
+	getLoadTimeSeconds: function() {
+		if(!window.performance || !window.performance.timing)return false;
+		return (window.performance.timing.domContentLoadedEventEnd - window.performance.timing.navigationStart) / 1000;
+	},
+
+	getPageName: function() {
+		return document.title ? document.title.split('-')[0] : '';
+	},
+
+	labelLoadTime: function(loadTime) {
+		with(this)
+			return 	loadTime < 1 ? labels.excellent : 
+						 	loadTime < 2 ? labels.veryGood : 
+						 	loadTime < 3 ? labels.acceptable : 
+						 	loadTime < 5 ? labels.improve : labels.fix;
```

Comment:
Why squash the last two categories into one label, rather than have up to five 5 and < up to 8? If we put them together should we not just make greater than 5 "fix"?

Reply:
Re-labelled, thanks